server/src/subscriber/subscriber.service.ts

```diff
@@ -183,5 +215,38 @@ export class SubscriberService {
     return validCustomFieldValues.includes(value as CustomFieldValue);
   }
 
+  /**
+   * Convert the uploaded subscriptions object into a format for Handlebars to use in the confirmation email
+   * @param {object} subscriptions - The set of CDs the user is subscribing to
+   * @returns {boolean}
+   */
+  private convertSubscriptionsToHandlebars(subscriptions: object) {
+    var handlebars = { "citywide": false, "boroughs": [] }
```

Comment:
Would it make the templating in Sendgrid much more difficult if `boroughs` was an object wit keys for each borough instead of an array? I'm just worried having it as an array might be bug-prone.

Reply:
As far as I'm aware, it's not possible - I would have done it that way if it were, but I don't think you're able to iterate over anything that's not an array.